src/pretix/base/models/organizer.py

```diff
@@ -395,3 +395,96 @@ def get_events_with_permission(self, permission, request=None):
             return self.get_events_with_any_permission()
         else:
             return self.team.organizer.events.none()
+
+
+class OrganizerBillingModel(models.Model):
+    """
+    Billing model - support billing information for organizer
+    """
+
+    organizer = models.ForeignKey(
+        "Organizer", on_delete=models.CASCADE, related_name="billing"
+    )
+
+    primary_contact_name = models.CharField(
+        max_length=255,
+        verbose_name=_("Primary Contact Name"),
+    )
+
+    primary_contact_email = models.EmailField(
+        max_length=255,
+        verbose_name=_("Primary Contact Email"),
+    )
+
+    company_or_organization_name = models.CharField(
+        max_length=255,
+        verbose_name=_("Company or Organization Name"),
+    )
+
+    address_line_1 = models.CharField(
+        max_length=255,
+        verbose_name=_("Address Line 1"),
+    )
+
+    address_line_2 = models.CharField(
+        max_length=255,
+        verbose_name=_("Address Line 2"),
+    )
+
+    city = models.CharField(
+        max_length=255,
+        verbose_name=_("City"),
+    )
+
+    zip_code = models.CharField(
+        max_length=255,
+        verbose_name=_("Zip Code"),
+    )
+
+    country = models.CharField(
+        max_length=255,
+        verbose_name=_("Country"),
+    )
+
+    preferred_language = models.CharField(
+        max_length=255,
+        verbose_name=_("Preferred Language"),
+    )
+
+    tax_id = models.CharField(
+        max_length=255,
+        verbose_name=_("Tax ID"),
+    )
+
+    payment_information = models.TextField(
+        verbose_name=_("Payment Information"),
+    ),
```

Comment:
The payment_information field in OrganizerBillingModel has an erroneous trailing comma in its tuple definition that needs to be removed